src/app/modules/reports/components/time-entries-table/time-entries-table.component.ts

```diff
@@ -83,4 +94,10 @@ export class TimeEntriesTableComponent implements OnInit, OnDestroy, AfterViewIn
     const regex = new RegExp('http*', 'g');
     return regex.test(uri);
   }
+
+  bodyExportOptions(data, row, column, node){
+    console.log(data);
```

Comment:
Please remove this line

Reply:
sorry for that. This line is deleted in the new commit.